### PR TITLE
added HTMLURL field  to IntegrationInstallation

### PR DESCRIPTION
**Pull request by liubinyi**
added HTMLURL to IntegrationInstallationEvent and IntegrationInstallationRepositoriesEvent #591

### CODE REVIEW

**Comment by liubinyi** [comment]
@gmlewis 
i've run those before i sent the pr 
go get ./...
golint
go generate github.com/google/go-github/...
go test github.com/google/go-github/...
go vet github.com/google/go-github/...
go fmt

and i got my go env set up. I am not sure why would CI failed..

**Comment by gmlewis** [comment]
After you ran `go generate`, did you see if files (specifically, `github-accessors.go`) were updated?
If so, you will need to add those changes to a commit on this PR.

**Comment by liubinyi** [comment]
ah included them in the pr. thanks @gmlewis

**Comment by shurcooL** [comment]
I've posted a question/concern in the issue https://github.com/google/go-github/issues/591#issuecomment-287125801. Once that's resolved, I can review this PR.

Side note, either the commit message or PR description of this PR should say "Fixes #591." or equivalent. See https://help.github.com/articles/closing-issues-via-commit-messages/ for details.

File: github/event_types.go [hunk 1]
```diff
@@ -175,6 +175,7 @@ type IntegrationInstallationEvent struct {
 	Action       *string       `json:"action,omitempty"`
 	Sender       *User         `json:"sender,omitempty"`
 	Installation *Installation `json:"installation,omitempty"`
+	HTMLURL      *string       `json:"html_url,omitempty"`
```
Comment (gmlewis) [inline github/event_types.go]: As discussed in #591, please move this line to the `type Installation struct` in `github/integration_installation.go`.
Thank you, @liubinyi!

Comment (liubinyi) [inline github/event_types.go]: @gmlewis updated.

**Comment by gmlewis** [comment]
Thanks, @shurcooL... will do.

**Comment by gmlewis** [comment]
Merging.

**Comment by liubinyi** [comment]
Thanks @gmlewis  and  @shurcooL for taking care of that. sorry i messed it up the 2nd time. I will be more careful with those in the future.

**Pull request merged**